migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc

```diff
@@ -1,11 +1,11 @@
 [id="planning-considerations-3-4"]
-= Planning considerations
+= Network considerations
 include::modules/common-attributes.adoc[]
 :context: planning-considerations-3-4
 
 toc::[]
 
-This section describes considerations and strategies for planning your migration from {product-title} 3 to 4.
+Review the strategies for redirecting your application network traffic after migration.
 
 include::modules/migration-DNS-considerations.adoc[leveloffset=+2]
```

Comment:
You have 3 modules included here but only 2 show up in the TOC.

Reply:
Four modules are included, but modules 2 and 3 are subsections of 1. Should they appear in the TOC?